pipelines/protocolB.yaml

```diff
@@ -0,0 +1,278 @@
+      doApplyGains: false
+      doFringe: false
+      doLinearize: parameters.linearize
+      doDefect: parameters.defects
+  cptDark:
+    class: lsst.cp.pipe.cpDarkTask.CpDarkTask
+    config:
+      connections.inputExp: 'cptDarkIsr'
+      connections.outputExp: 'cptDarkProc'
+  cptDarkCombine:
+    class: lsst.cp.pipe.cpCombine.CalibCombineTask
+    config:
+      connections.inputExpHandles: 'cptDarkProc'
+      connections.outputData: 'dark'
+      calibrationType: 'dark'
+      exposureScaling: "DarkTime"
+      python: config.mask.append("CR")
+
+  # Flat generation:
+  cptFlatIsr:
+    class: lsst.cp.testing.CptIsrTask
+    config:
+      expectedExposureType: 'flat'
+      connections.ccdExposure: 'raw'
+      connections.bias: 'bias'
+      connections.dark: 'dark'
+      connections.outputExposure: 'cptFlatProc'
+      doWrite: true
+      doBrighterFatter: false
+      doFlat: false
+      doFringe: false
+      doApplyGains: false
+      doLinearize: parameters.linearize
+      doDefect: parameters.defects
+  cptFlatMeasure:
+    class: lsst.cp.pipe.cpFlatNormTask.CpFlatMeasureTask
+    config:
+      connections.inputExp: 'cptFlatProc'
+      connections.outputStats: 'cptFlatStats'
+  cptFlatNorm:
+    class: lsst.cp.pipe.cpFlatNormTask.CpFlatNormalizationTask
+    config:
+      connections.inputMDs: 'cptFlatStats'
+      connections.outputScales: 'cptFlatNormScales'
+  cptFlatCombine:
+    class: lsst.cp.pipe.cpCombine.CalibCombineByFilterTask
+    config:
+      connections.inputExpHandles: 'cptFlatProc'
+      connections.inputScales: 'cptFlatNormScales'
+      connections.outputData: 'flat'
+      calibrationType: 'flat'
+      exposureScaling: InputList
+      scalingLevel: AMP
+
+  #high-value (bright pixels) defects
+  cptBrightDefectIsr:
+    class: lsst.cp.testing.CptIsrTask
+    config:
+      expectedExposureType: 'dark'
+      connections.ccdExposure: 'raw'
+      connections.outputExposure: 'cptBrightDefectsProc'
+      connections.bias: 'bias'
+      connections.dark: 'dark'
+      connections.flat: 'flat'
+      doWrite: true
+      doOverscan: true
+      doAssembleCcd: true
+      doBias: true
+      doVariance: false
+      doLinearize: parameters.linearize
+      doCrosstalk: parameters.crosstalk
+      doBrighterFatter: false
+      doDark: false
+      doStrayLight: false
+      doFlat: false
+      doFringe: false
+      doApplyGains: false
+      doDefect: parameters.defects
+      doSaturationInterpolation: false
+      growSaturationFootprintSize: 0
+  cptMeasureBrightDefects:
+    class: lsst.cp.pipe.defects.MeasureDefectsTask
+    config:
+      connections.inputExp: 'cptBrightDefectsProc'
+      connections.outputDefects: 'cptPartialBrightDefects'
+  cptMergeBrightDefects:
+    class: lsst.cp.pipe.defects.MergeDefectsTask
+    config:
+      connections.inputDefects: 'cptPartialBrightDefects'
+      connections.mergedDefects: 'brightDefects'
+
+  #low-value (dark pixels) defects
+  cptDefectIsr:
```

Comment:
Should this be DarkDefect?  Also, I thought we should be using the combined darks for the defects.

Reply:
As above, I'll need to fix this.  It won't be `DarkDefect`, as low/faint are measured on the flat and `dark` is already too overloaded in this context.